### DISCUSSION TITLE
help

### DISCUSSION THREAD

**Discussion by Agoni00** [NOAA-EMC/WW3#541] [Q&A]
I have a little puzzle about the three-dimensional radiation stress. The unit of the radiation stress term output by the mode is N/m. Why, shouldn't the unit of the stress term be N/(m*m)?
![sxy_in_ww3 199602](https://user-images.githubusercontent.com/92639723/142148828-bdd6ed49-f7e7-4e43-8bac-574dca02b148.png)

**Comment by aliabdolali** [comment]
@Agoni00 sxx, syy and syy are 2-dimensional radiation stress not 3D and the unit should be m2s−2 (N m−2/ρ).

**Reply by Agoni00** [reply]
@aliabdolali Thank you very much for your answer. Do you mean that the unit of radiation stress  (Sxx, Syy,sxy) should be N m−2 ?
